src/tests/Nightwatch/Tests/AddContent.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  '@tags': ['AddContent'],
+  addContentRenders(browser) {
+    browser
+      .logUserIn()
+      .relativeURL('/node/add')
+      .waitForElementVisible('[data-nightwatch="node-type-list"] a', 10000)
+      .getText(
```

Comment:
I'm wondering why you picked this way instead of something like: ```  browser.expect.element('[data-nightwatch="node-type-list"] a:nth-child(1) span:first-child').text.to.equal('Article');
``` ❓